docs/learn/digital-assets/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: '👋🏻 Getting Started'
+sidebar_position: 1
+description: Create a Hardhat project and start building your smart contracts for LUKSO.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started building contracts
+
+The LUKSO ecosystem offers smart contract developers a lot of [new standards](../../standards/introduction.md) and [tools](../../tools/getting-started.md) to build **powerful, modular, and standardized** blockchain applications.
+
+This page will guide you through setting up your development environment to build on LUKSO.
+
+If you need more low level information about our contracts, you can check the dedicated [contracts](../../contracts/introduction.md) section.
+
+### Playground Repository
+
+In the following guides, we will use the [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository with a full Hardhat setup, including ready-to-go network configurations, sample contracts, and scripts to deploy and verify LSP-based contracts on LUKSO.
+
+<div style={{textAlign: 'center'}}>
+
+<img
+src="/img/guides/playground_hardhat.png"
+alt="LUKSO Playground Hardhat"
+/>
+
+</div>
+
+## Clone the Playground Repository
+
+First, clone the playground repository and navigate into the smart-contracts-hardhat folder:
+
+```bash
+git clone https://github.com/lukso-network/lukso-playground.git
+cd lukso-playground/smart-contracts-hardhat
+```
+
+Once navigated, you have a working Hardhat setup.
+
+:::info
+
+**This repository uses the [bun](https://bun.sh) toolkit to manage packages and run scripts. Make sure to [install it](https://bun.sh/docs/installation) first.**
+
+:::
+
+To install all the dependencies, run the below command:
```

Comment:
Suggest putting bun first at the top and not put the info callout, but just as a step.